content/en/docs/instrumentation/java/exporters.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Exporters
+weight: 50
+---
+
+In order to visualize and analyze your traces, you will need to export them to a
+backend such as [Jaeger](https://www.jaegertracing.io/) or
+[Zipkin](https://zipkin.io/). OpenTelemetry Java provides exporters for some
+common open source backends.
+
+If you use the Javaagent for
```

Comment:
Should we refer to it as Javaagent or "Java agent"? I've seen this both ways, so unclear what's preferred

Reply:
[Time to resolve this issue](https://github.com/open-telemetry/opentelemetry.io/issues/1661)